src/zope/pagetemplate/tests/test_basictemplate.py

```diff
@@ -32,26 +30,25 @@ def tearDown(self):
 
     def test_if_in_var(self):
         # DTML test 1: if, in, and var:
-        pass # for unittest
-        """
-        %(comment)[ blah %(comment)]
-        <html><head><title>Test of documentation templates</title></head>
-        <body>
-        %(if args)[
-        <dl><dt>The arguments to this test program were:<p>
-        <dd>
-        <ul>
-        %(in args)[
-          <li>Argument number %(num)d was %(arg)s
-        %(in args)]
-        </ul></dl><p>
-        %(if args)]
-        %(else args)[
-        No arguments were given.<p>
-        %(else args)]
-        And thats da trooth.
-        </body></html>
-        """
+        # """
+        # %(comment)[ blah %(comment)]
+        # <html><head><title>Test of documentation templates</title></head>
+        # <body>
+        # %(if args)[
+        # <dl><dt>The arguments to this test program were:<p>
+        # <dd>
+        # <ul>
+        # %(in args)[
+        #   <li>Argument number %(num)d was %(arg)s
+        # %(in args)]
+        # </ul></dl><p>
+        # %(if args)]
+        # %(else args)[
+        # No arguments were given.<p>
+        # %(else args)]
+        # And thats da trooth.
+        # </body></html>
+        # """
```

Comment:
Why is this still here?

Reply:
Do you mean the triple quotes or the entire comment? The comment is basically a copy of the test file (dtml1.html) that I assume someone felt had value to repeat for reference. The triple quotes....well, we could either say they distinguish the "literal" content from the actual comment, or we could call them overlooked.

Reply:
I meant the entire comment. It is not clear that it is a copy of the file contents. When the file contents are changed it is not sure that it will be changed here, too. As modern editors allow to view more than one file at once I'd suggest to remove the entire comment. (I consider it as duplicate code.)